### ISSUE TITLE
Expansion Missing Samples

### ISSUE THREAD

**Issue by nytemairqt** [christophhart/HISE#253]
**Forum Discussion:** https://forum.hise.audio/topic/5121/confusing-convolution/34?_=1645050228056

**Commit:** [ecd5f153a7d47e335caf52f446b63cb7929598c7](https://github.com/christophhart/HISE/commit/ecd5f153a7d47e335caf52f446b63cb7929598c7)

**Conditions to reproduce the bug:**

OS: Windows (havent checked MacOS) 
HISE: yes
Compiled Project: yes

**Problem Description**

Using a basic expansion system, installing and loading an expansion in the compiled plugin results in a "Samples Folder Missing" popup loop

Barebones project with binary attached 

![image](https://user-images.githubusercontent.com/63175256/154368799-c4c341c2-d7bb-4315-baaf-ea96644981eb.png)

[neatplayerdebug.zip](https://github.com/christophhart/HISE/files/8084077/neatplayerdebug.zip)

**Comment by davidhealey** [comment]
> Just tested the latest commit but since I'm using Encoded expansion mode it hasn't changed, should I just bite the bullet and switch to Full Expansion mode? I'm not sure how that would affect my users having to re-download every expansion...

Full expansions is a totally different thing from the other expansion types and likely isn't what you need for your project. Also it's unrelated to this problem you're having.

Are the samples actually missing when this popup appears or is it just a false message?

**Comment by nytemairqt** [comment]
Clicking "ignore" lets you minimize the window but the samples don't play, the .ch1 file extracts to the right folder, and the linkwindows file is pointing to it correctly 

Re-opening the expansion gives the prompt every time, redirecting the plugin to the ch1 file doesn't solve it either

**Comment by davidhealey** [comment]
> Re-opening the expansion gives the prompt every time, redirecting the plugin to the ch1 file doesn't solve it either

Your example project works here. I installed the hr file using the button on your UI not using the prompt that appears on first run asking to install or locate the samples.

**Comment by nytemairqt** [comment]
That preprocessor helps get rid of the popup, but the samples still dont work. 

Can I send you the NEAT Player binary and the expansion I'm using to see if you can recreate it?

**Comment by davidhealey** [comment]
Sure, but isn't that what's in the download in the first post of this thread?

**Comment by nytemairqt** [comment]
The one above is a barebones example but neither worked for me, I'm not sure what's going on anymore lol

[NEAT Player Debug.zip](https://github.com/christophhart/HISE/files/8100987/NEAT.Player.Debug.zip)

**Comment by nytemairqt** [comment]
Yeah you're right, moving the linkwindows file gets it working. I'll make a forum post to keep the git organised :) Thanks
